libs/docs/platform/table/examples/platform-table-custom-title-example.component.html

```diff
@@ -0,0 +1,21 @@
+<fdp-table [dataSource]="source" [trackBy]="trackBy" emptyTableMessage="No data found">
```

Comment:
maybe we could just update one of the existing examples? table's page that long and takes that much time to load so i'm kinda against of extending it even more at least until we speed up it.